Docs: support bundle section for 9.5 What's New (#65641)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v9-5.md

### Organization roles for users logged in through Auth providers

_Generally available in all editions of Grafana._

> We are slowly rolling out a feature toggle for enforcing synchronization of organization roles for organization roles sync from the authentication provider on user sign-in. The feature will prohibit a user from changing organization roles synced with external auth providers.

Reviewer: Maybe: "This release comes with a feature toggle called `onlyExternalOrgRoleSync` that you can enable. Once enabled, users signing in to Grafana cannot change organization roles synchronized with an external authentication provider."

> If you want to enable this feature, you'll need to enable the `onlyExternalOrgRoleSync` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.

Reviewer: And then remove this first sentence.

> A reminder that there is a setting to prevent synchronization of organization roles from the authentication provider. Refer to [skip org role sync](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#authgrafana_com-skip_org_role_sync) for more information.

Author: @eleijonmarck could you please double-check these changes? The linter was complaining about spelling, so I read through this section and I think you can simplify the last paragraph a bit.

Reviewer 2: yea i like the shortening of the paragraph. thx for looking into it

> A reminder that there is a setting to prevent synchronization of organization roles from the authentication provider. Refer to [skip org role sync](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#authgrafana_com-skip_org_role_sync) for more information.

Reviewer: ```suggestion
You can also prevent the synchronization of organization roles from the authentication provider. For more information, refer to [skip org role sync](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#authgrafana_com-skip_org_role_sync).
```

## Support bundles

_Generally available in all editions of Grafana._

> Support bundles provide a simple way to collect information about your Grafana instance through Grafana's user interface. In a few clicks you can create a support bundle containing data about migrations, plugins, settings and more. This is particularly useful when troubleshooting issues with your Grafana instance.

Reviewer: How is creating a support bundle useful to the user? Do they do something with it, like send it to us so that we can troubleshoot?

> Support bundles provide a simple way to collect information about your Grafana instance through Grafana's user interface. In a few clicks you can create a support bundle containing data about migrations, plugins, settings and more. This is particularly useful when troubleshooting issues with your Grafana instance.

Reviewer: ```suggestion
Support bundles provide a simple way to collect information about your Grafana instance through Grafana's user interface. In a few clicks, you can create a support bundle containing data about migrations, plugins, settings, and more. This is particularly useful when troubleshooting issues with your Grafana instance.
```
